### ISSUE TITLE
Accessibility label is not visible for ios simulator assigned to textInput in appium inspector react native

### ISSUE THREAD

**Issue by chetnaag95** [facebook/react-native#37433]
### Description

Hi,

Accessibility ID is not visible in appium inspector on inspecting textInput in react native for ios simulator, but are visible on Button

Also the same accessible id are visible for android in appium inspector

Appium - v2.0.0-beta.66
WDA - v4.9.0
Node - v14.21.3
MacOS - Ventura 13.2
Xcode - 14.2
npm - 9.6.6

### React Native Version

0.71.0

### Output of `npx react-native info`

System:
    OS: macOS 13.2
    Memory: 22.28 MB / 32.00 GB
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 18.14.2 - ~/.nvm/versions/node/v18.14.2/bin/node
    Yarn: 1.22.19 - /usr/local/bin/yarn
    npm: 9.5.0 - ~/.nvm/versions/node/v18.14.2/bin/npm
    Watchman: 2023.05.08.00 - /usr/local/bin/watchman
  Managers:
    CocoaPods: 1.12.0 - /usr/local/bin/pod
  SDKs:
    iOS SDK:
      Platforms: DriverKit 22.2, iOS 16.2, macOS 13.1, tvOS 16.1, watchOS 9.1
    Android SDK: Not Found
  IDEs:
    Android Studio: 2022.1 AI-221.6008.13.2211.9514443
    Xcode: 14.2/14C18 - /usr/bin/xcodebuild
  Languages:
    Java: 11.0.18 - /usr/bin/javac
  npmPackages:
    @react-native-community/cli: Not Found
    react: Not Found
    react-native: ^0.71.0 => 0.71.0
    react-native-macos: Not Found
  npmGlobalPackages:
    *react-native*: Not Found

### Steps to reproduce

connected ios simulator to appium inspector. I am not able to view accessbility label for textInput in appium inspector which for button element and textInput accessbility label is visible for android as well.

### Snack, code example, screenshot, or link to a repository

<TextInput
accessible
accessibilityLabel={translateLabel(this.translate, 'general', 'field', 'email_phone')}
accessibilityHint="Navigates to the previous screen"
editable={XXX}
style={[this.state.style.username.field, alignStyle, { margin: 5 }]}
placeholder={this.translate('email_phone')}
value={this.state.username}
autoCapitalize="none"
autoCorrect={false}
spellCheck={false}
dataDetectorTypes="none"
placeholderTextColor={GREY}
underlineColorAndroid="transparent"
/>
